File: PHP/pertemuan11/nim_array.php
<?php
// Input NIM
$nim = readline("Masukkan NIM: ");

// Ubah string menjadi array angka
$digit = array_map('intval', str_split($nim));

// Total
$total = 0;
foreach($digit as $d){
    $total += $d;
}

// Maks
$maks = max($digit);

// Minim
$minim = min($digit);

// Rata-rata
$rata = $total / count($digit);

// Reverse array
$rev = array_reverse($digit);


echo "Digit        : " . implode(" ", $digit) . PHP_EOL;
echo "Total        : " . $total . PHP_EOL;
echo "Maks         : " . $maks . PHP_EOL;
echo "Minim        : " . $minim . PHP_EOL;
echo "Rata-rata    : " . $rata . PHP_EOL;
echo "Reverse      : " . implode(" ", $rev) . PHP_EOL;
?>

File: PYTHON/pertemuan11/nim_array.py
nim = input("Masukkan NIM: ")
# ubah sring menjadi array 
digit = [int(c) for c in nim]

#total
total = 0
for d in digit:
    total += d

#max
maks = digit[0]
for d in digit:
    if d > maks:
        maks = d

#min
minim = digit[0]
for d in digit:
    if d < minim:
        minim = d

#rata
rata = total / len(digit)

#reverse array
rev = []
for i in range(len(digit)-1, -1, -1):
    rev.append(digit[i])

print("digit:", digit)
print("total:", total)
print("maks:", maks)
print("minim:", minim)
print("rata-rata=", rata)
print("reverse:", rev)